### ISSUE TITLE
npm run test tries to load development.env from incorrect location in Windows

### ISSUE THREAD

**Issue by yncat** [colyseus/colyseus#421]
Your issue may already be reported!
Please search on the [issue tracker](../) before creating one.

## Steps to Reproduce (for bugs)

1. run `npm init colyseus-app testserver` to create an app from template.
2. cd to the app and run `npm run test` .
3. Get console output: "development.env not found"

## Context

I expected the test to load development.env from my_project_dir\development.env. However, it was actually trying to load from my_project_dir\node_modules\mocha\development.env.
I inserted console.log into node_modules\@colyseus\arena\build\index.js, and found out that

```
require?.main?.filename || process.cwd()
```

yielded

```
my_project_directory\node_modules\mocha\bin\mocha
```

which made the final envPath

```
my_project_dir\node_modules\mocha\development.env
```

https://github.com/colyseus/colyseus/blob/c47807e27203d6827003d705dd918ceae287b837/packages/arena/src/index.ts#L23

running npm start loads development.env from the correct location.

Is this an environment-specific issue?

## Your Environment

* Colyseus Version: 0.14.20
* Node.js Version: 15.14.0
* Operating System and version: Windows 10
* Shell: Windows Terminal
* Link to your project: not published yet

**Comment by endel** [comment]
Thank you for noticing and reporting @yncat! This has been fixed now in `@colyseus/arena@0.14.22`

**Comment by yncat** [comment]
@endel Thank you for the fix. It works like a charm now!
